### DISCUSSION TITLE
How to use paraview in OpenPNM

### DISCUSSION THREAD

**Discussion by GaoC-en** [PMEAL/OpenPNM#2355] [Q&A] [answered]
Dear professor:
I firstly used OpenPNM and finished the tutorial one. But I can not using the paraview to see the simulation results. After looking through the existed questiones, the former answer websites are not available now. Could you please show me the current website about using Paraview in OpenPNM. Sincerely thanks for your help.

**Comment by ma-sadeghi** [comment, marked as answer]
The website (and `openpnm`) are undergoing lots of changes, sorry about that. Meanwhile, see [here](https://github.com/PMEAL/OpenPNM/blob/dev/examples/getting_started/plotting_in_paraview.rst) and [here](https://github.com/PMEAL/OpenPNM/blob/dev/examples/getting_started/view_vtp_in_paraview.md). We can't promise, but maybe by the end of April, v3 will be out and the website will be updated accordingly.

**Reply by GaoC-en** [reply]
Thanks a lot for your helping!
